src/Microsoft.AspNetCore.Hosting.Abstractions/IWebHostBuilder.cs

```diff
@@ -33,6 +42,13 @@ public interface IWebHostBuilder
         IWebHostBuilder ConfigureServices(Action<IServiceCollection> configureServices);
```

Comment:
We should mark this as obsolete, same with `ConfigureLogging(Action<ILoggerFactory> configureLogging)`